scripts/run-bundled-codegen.sh

```diff
@@ -8,7 +8,7 @@ SCRIPT_DIR="$(dirname "$0")"
 
 # Get the SHASUM of the tarball
 ZIP_FILE="${SCRIPT_DIR}/apollo.tar.gz"
-SHASUM="$(/usr/bin/shasum -a 256 "${ZIP_FILE}")"
+SHASUM="$(/usr/bin/shasum -a 256 "${ZIP_FILE}" | /usr/bin/awk '{ print $1 }')"
```

Comment:
Not super-familiar with `awk` - why `$1` instead of `$0`?

Reply:
`$0` would print the full input, `$1` prints the shasum, and `$2` would print the filename.